memcache/memcache.go

```diff
@@ -644,6 +669,11 @@ func (c *Client) DeleteAll() error {
 	})
 }
 
+// Ping checks if instances are alive.
```

Comment:
This doesn't make it clear whether it checks requires any or all servers to be up to return a nil error.